### PR TITLE
fix(delete files on disk): add delete multiple file delete

### PR DESCRIPTION
**Pull request by jonathangoulding**

### CODE REVIEW

File: packages/forms-web-app/src/controllers/examination/file-upload/fileManagement.js [hunk 1]
```diff
@@ -22,8 +22,16 @@ const deleteFileOnDisk = async (file) => {
 	await fs.unlinkSync(fileToDeletePath);
 };
 
+const iterateDeleteFileOnDisk = async (files) => {
+	for (const filesKey of files) {
+		const fileToDeletePath = fileUploadPath(filesKey.uniqueFileName);
+		await fs.unlinkSync(fileToDeletePath);
```
Comment (jonathangoulding) [inline packages/forms-web-app/src/controllers/examination/file-upload/fileManagement.js:28]: No real need to have this as sync other than to allow us to throw an error if the delete fails.

**Review by jonathangoulding — COMMENTED** [review]
Mentioned use of sync

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**